prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -407,6 +409,14 @@ class ProcessData : CliktCommand(
             else -> listOf(Pair(inputReport, getOutputFormat(Report.Format.CSV)))
         }
 
+        if (warnings.size > 0) {
+            echo("Problems occurred during translation to output schema:")
+            warnings.forEach {
+                echo("${it.scope} ${it.id}: ${it.details}")
+            }
+            echo()
+        }
+
```

Comment:
Similar to the reports endpoint, if you use `--route` and a failure occurs, it will continue to do other translations.